easybuild/easyblocks/w/wrf.py

```diff
@@ -98,20 +98,26 @@ def configure_step(self):
         # HDF5 (optional) dependency
         hdf5 = get_software_root('HDF5')
         if hdf5:
+            env.setvar('HDF5', hdf5)
             # check if this is parallel HDF5
             phdf5_bins = ['h5pcc', 'ph5diff']
             parallel_hdf5 = True
             for f in phdf5_bins:
                 if not os.path.exists(os.path.join(hdf5, 'bin', f)):
                     parallel_hdf5 = False
                     break
-            if not (hdf5 or parallel_hdf5):
-                raise EasyBuildError("Parallel HDF5 module not loaded?")
-            else:
+            if parallel_hdf5:
                 env.setvar('PHDF5', hdf5)
+            else:
+                self.log.info("Parallel HDF5 module not loaded, assuming that's OK...")
         else:
             self.log.info("HDF5 module not loaded, assuming that's OK...")
 
+        # Parallel netCDF (optional) dependency
+        pnetcdf = get_software_root('PnetCDF')
```

Comment:
@andreas-h We don't have any easyconfigs for `PnetCDF` yet, I think... Is this an entirely separate package from `netCDF`, or does it just require `netCDF` to be installed with specific configure options?